src/meta-srv/src/metasrv/builder.rs

```diff
@@ -311,32 +312,33 @@ impl MetasrvBuilder {
         ));
         region_migration_manager.try_start()?;
 
+        if !is_remote_wal && options.enable_region_failover {
+            warn!("Region failover is not supported in the local WAL implementation!");
```

Comment:
Should we terminate process when local wal is used and region failover is enabled instead of warning?